src/renderer/components/settings-general-appearance.tsx

```diff
@@ -193,13 +224,47 @@ export class AppearanceSettings extends React.Component<
             </a>. The easiest way to get started is to clone one of the two existing
             themes and to add your own colors.
           </p>
+          <p>
+            Additionally, if you wish to import a Monaco Editor theme, just pick your JSON file and Fiddle will do the rest.
+          </p>
           <Button
             onClick={this.createNewThemeFromCurrent}
             text='Create theme from current selection'
             icon='duplicate'
           />
+          <Button
+            onClick={this.createNewThemeFromMonaco}
+            text='Add a Monaco Editor theme'
+            icon='duplicate'
+          />
         </Callout>
       </div>
     );
   }
+
+  private async promptForTheme(defaultTheme: LoadedFiddleTheme) {
+    const filePicked = await remote.dialog.showOpenDialog({
```

Comment:
The `remote` object might be going away soon (possibly), so we should try to avoid using it in new code. Could you find a way to do this without the `dialog` module at all? Maybe just with a native HTML5 file dialog? From the looks of it, the HTML5 version would do everything you need: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file